signatures/gemss/gemss.yaml

```diff
@@ -0,0 +1,19 @@
+name: 
+category: multivariate
+year:
+	paper: 2017-09-23
+	candidate: 2017
```

Comment:
There are dashes missing here.

Reply:
No, it's a dictionary. With dashes it would mean `year = [ { paper: 2017-09-23 }, {candidate: 2017} ]`

Reply:
You're right, my mistake. However, the validation fails here:

```
signatures/gemss/gemss.yaml
[
  {
    keyword: 'additionalProperties',
    dataPath: '',
    schemaPath: '#/additionalProperties',
    params: { additionalProperty: 'paper' },
    message: 'should NOT have additional properties'
  }
]
```
The paper property should be an integer and there must be something wrong with the whitespaces.

Reply:
The file dbscheme-proposal.md allows month and day, but then the date is no integer.

Reply:
That's right. I'd prefer to allow both years and dates, but it'd make things complicated in the backend... What do you think @benj-zen ?

Reply:
I would stay with years for now.